lua/lib/reflection.lua

```diff
@@ -212,12 +228,13 @@ function reflection:begin_playback()
           if self.loop == 1 then
             self.step = 0
             self:_clear_flags()
-            self:start_callback()
+            queued_start_callback = true
           end
         end
       end
     else
```

Comment:
might be nice to add a little comment here saying
`-- not first pass; we'll process`
or something similar; this helps me parse the other comment in this function better